submission/submission-webapp/src/main/resources/application.yml

```diff
@@ -65,6 +65,7 @@ app:
   requestFilesPath: # Base directory absolute path where request file list are stored.
   fireTempDirPath: # Absolute path of the temp folder for FIRE storage
   instanceBaseUrl: # Instance base url e.g. http://server.ebi.ac.uk:5541
+  subBasePath: # Extra path added as a prefix of submission relative path. Allows to allocate submissions under different paths
```

Comment:
I'd suggest:
```If present, it will be used as a prefix to the submission relative path```

Not sure about the purpose part. Usually, you want to document what something does, not what's the purpose since, in reality, we're not actually checking that